### PR TITLE
Login Flow

### PR DESCRIPTION
**Pull request by pandilladivya**
Once the user logs in to the application, Login page is not shown to the user again until they logout or uninstalls the application.

If the user skips sign in, user is redirected to the homepage and login page is shown again when the user reopens the app / reruns the app. ( this will be changed in future at the time of event registrations )

@HimanshiJain @sandarumk 
Thank you.

### CODE REVIEW

**Comment by HimanshiJain** [comment]
@pandilladivya What happens with Facebook and Google sign in?after signing in with Facebook/Google once, I am not able to see the login screen skipped.

File: app/src/main/java/vola/systers/com/android/activities/SignInActivity.java [hunk 1]
```diff
@@ -238,8 +251,7 @@ private void handleGoogleSignInResult(GoogleSignInResult result) {
             Log.i(TAG, "Name: " + personName + ", email: " + email
                     + ", Image: " +personPhotoUrl);
 
-            Intent intent = new Intent(SignInActivity.this,MenuActivity.class);
-            startActivity(intent);
+            launchHomeScreen();
```
Comment (pandilladivya) [inline app/src/main/java/vola/systers/com/android/activities/SignInActivity.java]: Here in Google Sign In, If the signIn is successful I am calling launchHomeScreen() method, where in we are handling this case. And I missed to handle fb signIn, will add. :)

File: app/src/main/java/vola/systers/com/android/manager/PrefManager.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+public class PrefManager {
+    SharedPreferences pref;
+    SharedPreferences.Editor editor;
+    Context _context;
+
+    // shared pref mode
+    int PRIVATE_MODE = 0;
+
+    // Shared preferences file name
+    private static final String PREF_NAME = "MyPrefs";
```
Comment (pandilladivya) [inline app/src/main/java/vola/systers/com/android/manager/PrefManager.java]: Yeah. Will rename 👍

Comment (sandarumk) [inline app/src/main/java/vola/systers/com/android/manager/PrefManager.java]: Is this done?

Comment (pandilladivya) [inline app/src/main/java/vola/systers/com/android/manager/PrefManager.java]: Yes. Done now. Amended changes to my previous commit! :)

**Pull request merged**